setup.py

```diff
@@ -131,6 +131,7 @@
           'sqlparse>=0.2.4',
           'word2number>=1.1',
           'pytorch-pretrained-bert>=0.6.0',
+          'pytorch-transformers @ https://api.github.com/repos/huggingface/pytorch-transformers/tarball/a7b4cfe9194bf93c7044a42c9f1281260ce6279e',
```

Comment:
Hi @matt-gardner, why not use `pytorch-transformers==1.1.0`?

PyPI won't allow direct URL like this one, as discussed in https://github.com/pypa/pip/issues/6301
And error msg should be raised as followed when running `twine upload`
```
HTTPError: 400 Client Error: Invalid value for requires_dist. Error: Can't have direct dependency: 'pytorch-transformers @ https://api.github.com/repos/huggingface/pytorch-transformers/tarball/a7b4cfe9194bf93c7044a42c9f1281260ce6279e' for url: https://upload.pypi.org/legacy/
```

Reply:
Because the auto stuff hadn't been released when I wrote this. It has been now, so this can be updated. PR welcome.